scripts/utils.mjs

```diff
@@ -133,4 +135,38 @@ function sortArray(original, {key=null, lowToHigh=true}={}){
     const y = foundry.utils.getProperty(b, key);
     return _sortFunction(x,y,lowToHigh)
   });
+}
+
+/**
+ * Function that changes "/my/awesome/filepath/epic-image.png" to "Epic Image",
+ * or "/my/awesome/filepath/epic_sound.ogg" to "Epic Sound",
+ * or "/my/awesome/filepath/epic%20video.webm" to "Epic Video"
+ * By simply wrapping FoundryVTT's AudioHelper.getDefaultSoundName(path) function, 
+ * which confuses people to think it is just for sounds.
+ * @param {string} path                       Path of image/video/sound etc.
+ * @returns {string}                          The changed name.
+ */
+function getDefaultName(path){
+  return AudioHelper.getDefaultSoundName(path);
+}
+
+/**
+ * Function that broadcasts audio to a user or a group of users.
+ * @param {string} src                        Path string of the sound to broadcast.
+ * @param {Array<user,string>} [users=[]]     Array of users or user ids.
+ * @param {number} [volume=1]                 Volume value between: 0-1.
+ * @param {boolean} [loop=true]               Loop or not, default false.
+ *  
+ */
+function playSoundForOthers(src, users=[], {volume=1,loop=false}={}) {
+  const ids = users.reduce((acc,u) => {
+    const id = u instanceof User ? u.id : game.users.has(u) ? u : null;
+    if(id) acc.push(id);
+    return acc;
+  },[]);
+  if(ids.length) {
+    game.socket.emit("playAudio", {src, volume, loop}, {recipients: ids});
+    ui.notifications.info(`sound: ${src} is playing for selected users.`);
```

Comment:
```suggestion
    ui.notifications.info(`Playing '${getDefaultName(src)}' for selected users.`);
```

Reply:
Fixed the changes... and for the toaster messages (/ error messages?) we might want to setup the language files.

Reply:
> Fixed the changes... and for the toaster messages (/ error messages?) we might want to setup the language files.

Agreed, this is best to do as they crop up, rather than later.